staging/src/k8s.io/client-go/tools/portforward/portforward.go

```diff
@@ -398,6 +402,9 @@ func (pf *PortForwarder) handleConnection(conn net.Conn, port ForwardedPort) {
 	err = <-errorChan
 	if err != nil {
 		runtime.HandleError(err)
```

Comment:
Let's remove this error handler since it results in the error being printed twice.